File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1035.cs
using System;

class URI
{
    static void Main() {
        string an = System.Console.ReadLine().Trim();
        string[] n = an.Split(' ');
        int a = int.Parse(n[0]), b = int.Parse(n[1]), c = int.Parse(n[2]), d = int.Parse(n[3]);
        
        if(b>c && d>a && c+d>a+b && c>0 && d>0 && a%2==0){ 
            Console.Write("Valores aceitos\n");
        }
        else{ 
            Console.Write("Valores nao aceitos\n");
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1035.lua
a = io.read("*number")
b = io.read("*number")
c = io.read("*number")
d = io.read("*number")

if b>c and d>a and c+d>a+b and c>0 and d>0 and a%2==0 then
    print("Valores aceitos")
else
    print("Valores nao aceitos")
end